lib/ansible/modules/windows/win_webpicmd.py

```diff
@@ -33,11 +33,11 @@
 short_description: Installs packages using Web Platform Installer command-line
 description:
     - Installs packages using Web Platform Installer command-line
-      (http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release).
-    - Must be installed and present in PATH (see win_chocolatey module; 'webpicmd' is the package name, and you must install 'lessmsi' first too)
-    - Install IIS first (see win_feature module)
+      (U(http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)).
```

Comment:
Fine by me, I was just making sure links were real links, and modules were real modules in the documentation ;-)